cdm/src/main/java/ucar/nc2/ft2/coverage/adapter/SimpleGeometryCSBuilder.java

```diff
@@ -227,19 +239,6 @@ public String toString() {
     f2.format("%s", type == null ? "" : type.toString());
     if (type == null) return f2.toString();
 
```

Comment:
Good call on keeping the `toString` up-to-date.

Reply:
Thanks! Might have to add some more to that in the future though...